tests/frontier/precompiles/test_precompiles.py

```diff
@@ -0,0 +1,99 @@
+import pytest
+
+from ethereum_test_tools import (
+    Account,
+    Alloc,
+    Environment,
+    StateTestFiller,
+    Transaction,
+)
+from ethereum_test_tools.code.generators import Conditional
+from ethereum_test_tools.vm.opcode import Opcodes as Op
+
+
+def precompile_addresses(fork: str) -> list[tuple[str, bool]]:
+    """
+    Return the precompile addresses for the given fork.
+
+    Args:
+        fork (str): The fork name.
+
+    Returns:
+        list[tuple[str, bool]]: The precompile addresses and whether they exist.
+
+    """
+    return [
+        ("01", True),
```

Comment:
you can access fork class from the test if you add  fixture in the parameter list.
fork: Fork 

and then it has a function that returns all precompiles currently active on that fork as integer list.